README.md

```diff
@@ -142,6 +145,54 @@ Below is a comprehensive list of available configuration properties.
 | webhook.projects.<_projectId_>.secret             | N/A                                             | Webhook secret used to validate webhook requests originating from the respective projectId                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | webhook.projects.<_projectId_>.skipSignatureCheck | N/A                                             | Boolean to indicate whether the signature should be validated. TODO remove in favor of empty secret.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 
+### CMAB Configuration Examples
+
+**Complete CMAB Configuration (OPTIMIZELY_CMAB)**:
+```json
+{
+  "enabled": true,
+  "predictionEndpoint": "https://custom-endpoint.com",
+  "requestTimeout": "5s",
+  "cache": {
+    "type": "redis",
+    "size": 2000,
+    "ttl": "45m",
+    "redis": {
+      "host": "redis:6379",
+      "password": "",
+      "database": 0
+    }
+  },
+  "retryConfig": {
+    "maxRetries": 3,
+    "initialBackoff": "100ms",
+    "maxBackoff": "10s",
+    "backoffMultiplier": 2.0
+  }
+}
+
+CMAB Cache Configuration (OPTIMIZELY_CMAB_CACHE):
+
+{
```

Comment:
Why do we need separate sections for CMAB Cache and Retry?  Wondering if we can use the one "CMAB" config block.